fundamentals.js

```diff
@@ -3,29 +3,45 @@
 // #1: Create an array of strings called `foods` that contains three foods.
 // Type your solution immediately below this line:
 
-
+var foods = ["orange", "grape", "steak"]
 
 // #2: Access the last item in the array and assign to a variable called `last`.
 // Type your solution immediately below this line:
 
+var last = foods[foods.length - 1] 
 
+//Come back to this 
 
 // #3: Create an empty array called `favoriteFoods`.
 // Type your solution immediately below this line:
 
-
+var favoriteFoods = []
 
 // #4: Create a `for` loop that adds each string in `foods` to `favoriteFoods`.
 // Type your solution immediately below this line:
 
-
+for(i=0; i<foods.length; i++){
+    favoriteFoods.push(foods)
```

Comment:
Close! You need to use the `i` variable to push each item in the `foods` array with `foods[i]`.

So something like:

```js
for (let i = 0; i < foods.length; i++) {
  favoriteFoods.push(foods[i]);
}
```

This will _technically_ work without a variable keyword before `i` in the for loop, and while the prompts say to use `var` for everything [`let` is better in for loops](https://wesbos.com/for-of-es6/) in particular.